xblock/mixins.py

```diff
@@ -242,6 +249,13 @@ def _field_data(self, field_data):
         warnings.warn("Setting _field_data is deprecated", FieldDataDeprecationWarning, stacklevel=2)
         self._deprecated_per_instance_field_data = field_data
 
+    @property
+    def supports_save(self):
+        """
+        Returns true if this block supports being saved.
+        """
+        return True
```

Comment:
I was thinking about this too, but it is a bit more complicated because `block.save()` is called 3 times in the runtime, and `aside.save()` is also called twice.  

One of the other `block.save()` calls is in `handle`, which I'm guessing applies to UIBlocks, but I'm not certain.  This code would need to change if we want the runtime to be less concerned with the need to save.  

The final call to `block.save()` is in `_usage_id_from_node` inside the runtime.  If we really want more independence between the mixin and runtime, this might require an even deeper change.

Additionally, the `aside.save()` calls could be ignored because they are in aside-specific runtime methods, or we could again try to match models of separating concerns out of the runtime.

Maybe I shouldn't have, but due to this I decided to punt and request that save just be added as a no-op.